tests/jsx-text-wrap/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -188,8 +197,9 @@ function DiffOverview(props) {
       <div className="alert alert-info">
         <p>
           This diff overview is computed against the current list of records in
-          this collection and the list it contained on <b>{humanDate(since)}</b>
-          .
+          this collection and the list it contained on <b>
+            {humanDate(since)}
+          </b>.
```

Comment:
This PR looks great to me. Do you feel this is a blocker? I don't. We could always improve things in another PR.

Btw, I this would be a pretty nice formatting:

```
this collection and the list it contained on 
{" "}<b>{humanDate(since)}</b>.
```